hack/lib/cpu.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
+
+import os
+
+ncpu = len(os.sched_getaffinity(0)) \
+    if hasattr(os, "sched_getaffinity") else os.cpu_count()
```

Comment:
maybe `go env GOMAXPROC` instead (default logic)

Reply:
`go env GOMAXPROCS` doesn't print anything @BenTheElder

Reply:
right, so if we fail to detect the preferred method then we get either `""` or the existing env value

we could instead read `os.Getenv("GOMAXPROC")` or just error here and have the caller detect and stick to default

Reply:
dropping the python stuff